build_daemon/lib/src/server.dart

```diff
@@ -83,6 +89,10 @@ class Server {
               ? _buildTargetManager.targets
               : _buildTargetManager.targetsForChanges(changes);
           await _build(targets, changes);
+        } else if (request is ContinueRequest) {
+          if (!_stepCompleter.isCompleted) {
```

Comment:
I do think we need to do one of two things here:

- Wait for all connected clients to issue a continue request
- Only allow one client to connect in this mode

Reply:
I referenced this in the PR description. Long term I do think we'll leverage this feature with Blaze so this will have to be addressed but I'm not sure it's worth concerning ourselves with at the moment. If you feel strongly I'll probably just add in the logic to wait for all connected clients.

Reply:
I do think we need to implement one of these options. Otherwise this is incurring significant technical debt.